assets/src/pages/room/RoomPage.tsx

```diff
@@ -115,7 +115,7 @@ const RoomPage: FC<Props> = ({ roomId, displayName, isSimulcastOn, manualMode, a
               className="focus:outline-none focus:shadow-outline m-1 w-full rounded bg-brand-grey-80 py-2 px-4 text-white hover:bg-brand-grey-100"
               type="submit"
             >
-              Show simulcast controls
+              Nerd options
```

Comment:
To me, it sounds a bit too unprofessional. I'd rather use something like `Advanced options`, `Internal options` or `Dev options`

Reply:
Okay :cry: